plugins/coreboot/README.md

```diff
@@ -0,0 +1,30 @@
+coreboot
+========
+
+WIP
+------------
+Depends on https://github.com/9elements/linux/tree/google_firmware_fmap2
+Depends on https://review.coreboot.org/c/coreboot/+/35377
```

Comment:
Since there is a dependency chain, what's the order?  I guess start with coreboot, then kernel, then fwupd stuff?

Reply:
yes